README.md

```diff
@@ -21,6 +21,12 @@ as terraform seems to have trouble otherwise.
 Speaking of which, copy the variable.tf_TEMPLATE to variable.tf, and modify as necessary.
 Certainly change the credentials file path, and the subnets if you want to.
 
+Validate the terraform will work as you expect
```

Comment:
I want to hook this up to CI at some point, but this is certainly better than no checking.